src/docs/admin/references/NodeSourcePolicyReference.adoc

```diff
@@ -299,9 +299,27 @@ This policy will never acquire more nodes than the infrastructure is capable of
 Symmetrically, if an infrastructure defines a minimum number of nodes to acquire, this policy will never go below this number.
 
 The DynamicPolicy will scale up when pending tasks are detected in the ProActive Scheduler queue and the Node Source does not have enough free nodes to assign.
+
 In some cases, the existing free nodes will not match the requirements of these pending tasks. This can happen when the task contains a <<../user/ProActiveUserGuide.adoc#_selection_scripts,Selection Script>> or a <<../user/ProActiveUserGuide.adoc#_generic_information_selection,NODE_ACCESS_TOKEN>> restriction that discards existing free nodes.
 In that case, the Dynamic Policy will use a *best-effort strategy* to handle these tasks that remain in pending state. When tasks remain in pending state longer than `initDelay` milliseconds, the node source will trigger a scale-up (one new node per pending task).
 
+The *loadFactor* is the main parameter that allows to control the scale up of the DynamicPolicy. It decides how many new ProActive Nodes will be deployed when considering the current number of pending and running tasks. *loadFactor* is a ratio which computes the expected number of Nodes in the Node Source according to the formula:
```

Comment:
to control the scale up --> to control the scale up/down ?

Reply:
ok